modules/about-redhat-openshift-gitops.adoc

```diff
@@ -0,0 +1,26 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/understanding-openshift-gitops.adoc
+
+[id="about-redhat-openshift-gitops_{context}"]
+= About {gitops-title}
+
+{gitops-title} ensures consistency in applications when you deploy them to different clusters in different environments, like: development, staging, and production. {gitops-title} organizes the deployment process around the configuration repositories and makes them the central element. It always has at least two repositories:
+
+  . Application repository with the source code
+  . Environment configuration repository that defines the desired state of the application
+
+These repositories contain a declarative description of the infrastructure you need in your specified environment. They also contain an automated process to make your environment match the described state.
+
+{gitops-title} uses Argo CD to maintain cluster resources. Argo CD is an open-source declarative tool for the continuous integration and continuous deployment (CI/CD) of applications. {gitops-title} implements Argo CD as a controller so that it continuously monitors application definitions and configurations defined in a Git repository. Then, Argo CD compares the specified state of these configurations with their live state on the cluster. Argo CD reports any configurations that deviate from their specified state. These reports allow administrators to automatically or manually resync configurations to the defined state. Therefore, Argo CD enables you to deliver global custom resources, like the resources that are used to configure OpenShift Container Platform clusters.
```

Comment:
I assume the spelling of "Argo CD" here (with a space) is correct, though I do see in https://docs.openshift.com/container-platform/4.6/architecture/argocd.html it's spelled (probably incorrectly) mashed-up as "ArgoCD".

Reply:
This paragraph ends up kinda long, maybe consider breaking it up either starting at "{gitops-title} implements Argo CD" or "These reports allow"?